traverson4j-hc5/src/main/java/uk/co/autotrader/traverson/http/ApacheHttpUriConverter.java

```diff
@@ -48,8 +49,8 @@ public HttpUriRequest toRequest(Request request) {
     public <T> Response<T> toResponse(CloseableHttpResponse httpResponse, URI requestUri, Class<T> returnType) throws IOException {
         Response<T> response = new Response<T>();
         response.setUri(requestUri);
-        response.setStatusCode(httpResponse.getStatusLine().getStatusCode());
-        for (Header responseHeader : httpResponse.getAllHeaders()) {
+        response.setStatusCode(httpResponse.getCode());
+        for (Header responseHeader : httpResponse.getHeaders()) {
```

Comment:
Whats the difference between getAllHeaders and getHeaders?

Reply:
getAllHeaders method has been removed in httpclient5. This is a helper method in the CloseableHttpResponse class designed to be backwards compatible with httpclient v4.x